backend/report_submission/views.py

```diff
@@ -453,3 +479,119 @@ def post(self, request, *args, **kwargs):
+                "view_id": "delete-additional-ueis",
+                "section_name": SN[NC.ADDITIONAL_UEIS].friendly_title,
+                "field_name": SN[NC.ADDITIONAL_UEIS].snake_case,
+                "form_section": FORM_SECTIONS.ADDITIONAL_UEIS,
+                "event_type": SubmissionEvent.EventType.ADDITIONAL_UEIS_DELETED,
+            },
+            "delete-secondary-auditors": {
+                "view_id": "delete-secondary-auditors",
+                "section_name": SN[NC.SECONDARY_AUDITORS].friendly_title,
+                "field_name": SN[NC.SECONDARY_AUDITORS].snake_case,
+                "form_section": FORM_SECTIONS.SECONDARY_AUDITORS,
+                "event_type": SubmissionEvent.EventType.SECONDARY_AUDITORS_DELETED,
+            },
+            "delete-additional-eins": {
+                "view_id": "delete-additional-eins",
+                "section_name": SN[NC.ADDITIONAL_EINS].friendly_title,
+                "field_name": SN[NC.ADDITIONAL_EINS].snake_case,
+                "form_section": FORM_SECTIONS.ADDITIONAL_EINS,
+                "event_type": SubmissionEvent.EventType.ADDITIONAL_EINS_DELETED,
+            },
+        }
+
+    def get(self, request, *args, **kwargs):
+        report_id = kwargs["report_id"]
+
+        try:
+            sac = SingleAuditChecklist.objects.get(report_id=report_id)
+
+            accesses = Access.objects.filter(sac=sac, user=request.user)
+            if not accesses:
+                raise PermissionDenied("You do not have access to this audit.")
+
+            # Context for every upload page
+            context = {
+                "report_id": report_id,
+            }
+            # Using the current URL, append page specific context
+            path_name = request.path.split("/")[2]
+
+            context["view_id"] = self.additional_context[path_name]["view_id"]
+            context["section_name"] = self.additional_context[path_name]["section_name"]
+
+            return render(request, "report_submission/delete-file-page.html", context)
+        except SingleAuditChecklist.DoesNotExist:
+            raise PermissionDenied("You do not have access to this audit.")
+
+    def post(self, request, *args, **kwargs):
+        report_id = kwargs["report_id"]
+        try:
+            sac = SingleAuditChecklist.objects.get(report_id=report_id)
+            accesses = Access.objects.filter(sac=sac, user=request.user)
+            if not accesses:
+                messages.error(request, "You do not have access to this audit.")
+                return redirect(request.path)
+            path_name = request.path.split("/")[2]
+            section = self.additional_context[path_name]
+
+            try:
+                excel_files = ExcelFile.objects.filter(
+                    sac=sac, form_section=section["form_section"]
+                )
+                logger.info(f"Deleting {excel_files.count()} files.")
+                excel_files.delete()
```

Comment:
Deleting the `ExcelFile` record does not, it turns out, delete the actual file from S3. The data is correctly nulled out in the `SingleAuditChecklist` record, and the `ExcelFile` record is gone, but the S3 file remains (see minio screenshot after deleting the CAP upload):
![image](https://github.com/GSA-TTS/FAC/assets/1425377/21866f21-cc66-4b75-a6ce-e0f02db5da92)

Reply:
Yes, this is what we discussed during dev huddle last week and my understanding was that we are not removing from s3 as part of this PR but we'll have another task/ticket to handle the deletion as part dissemination (and possibly clean up files for the already disseminated records).

Reply:
Right- couldn't 100% recall where we landed last week so I wanted to be sure - thanks for the clarification!